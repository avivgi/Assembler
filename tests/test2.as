.define sz = 2
.define abc = 4444
.define szff = 2
.define ab44c = 4444
.define l6grfgdfen = 4
.define len = 4
.define sz1= 2
.define abc1=4444
.define szff2=2
.define ab44c2 =             4444           
.define l6grfgdfen2 = 4
.define len2 = 4
.define 3LLL = 3
.define 3AAA=3;
.define A1=ddd
.define sz = 2

