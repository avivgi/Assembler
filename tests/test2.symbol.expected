Symbol name: sz        	Value: 2	type: 1: MDEFINE
Symbol name: abc       	Value: 4444	type: 1: MDEFINE
Symbol name: szff      	Value: 2	type: 1: MDEFINE
Symbol name: ab44c     	Value: 4444	type: 1: MDEFINE
Symbol name: l6grfgdfen	Value: 4	type: 1: MDEFINE
Symbol name: len       	Value: 4	type: 1: MDEFINE
Symbol name: sz1       	Value: 2	type: 1: MDEFINE
Symbol name: abc1      	Value: 4444	type: 1: MDEFINE
Symbol name: szff2     	Value: 2	type: 1: MDEFINE
Symbol name: ab44c2    	Value: 4444	type: 1: MDEFINE
Symbol name: l6grfgdfen2	Value: 4	type: 1: MDEFINE
Symbol name: len2      	Value: 4	type: 1: MDEFINE
